bin/nanocl/src/main.rs

```diff
@@ -817,6 +818,11 @@ mod tests {
     assert_cli_ok!("vm", "image", "rm", "-y", "test-cli-image");
   }
 
+  #[ntex::test]
+  async fn stats() {
```

Comment:
have you run the test? Because i think it will be locking there.. You need to execute it in a background task

Reply:
Fixed it